### PR TITLE
OCP4 STIG: Implement controls related to CM-7

### PR DESCRIPTION
**Pull request by jhrozek**
#### Description:

- OCP4 STIG: SRG-APP-000141-CTR-000320: Fix typo in attribute
- OCP4 STIG: SRG-APP-000142-CTR-000330 is implemented via several manual rules
- OCP4 STIG: SRG-APP-000383-CTR-000910 is inherently met
- OCP4 STIG: SRG-APP-000384-CTR-000915 is covered by CVO
- OCP4 STIG: SRG-APP-000386-CTR-000920 is inherently met

#### Rationale:

- Covers several SRGs related to NIST control CM-7

#### Review Hints:

- I will post the link to the SRG export later.

### CODE REVIEW

**Comment by jhrozek** [comment]
rebased on top of current master and ensured that all files touched have a newline at the end to conform with the project's coding standards

**Comment by jhrozek** [comment]
Oops, fixed a build issue and rebased on master.

**Comment by jhrozek** [comment]
For easier review, check out the [SRG export](https://github.com/ComplianceAsCode/content/suites/11248727916/artifacts/576196914)

**Comment by jhrozek** [comment]
@Vincent056 @rhmdnd PTAL

**Comment by jhrozek** [comment]
Not sure what these failures are:
```
make[3]: *** [eks/CMakeFiles/generate-internal-eks-cpe-oval-unlinked.xml.dir/build.make:73: eks/cpe-oval-unlinked.xml] Error 1
make[3]: *** Deleting file 'eks/cpe-oval-unlinked.xml'
make[2]: *** [CMakeFiles/Makefile2:614: eks/CMakeFiles/generate-internal-eks-cpe-oval-unlinked.xml.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs.... 
```
I'll just try rebasing..

**Comment by jhrozek** [comment]
timeouts and cluster install errors
/retest

File: controls/srg_ctr/SRG-APP-000386-CTR-000920.yml [hunk 1]
```diff
@@ -6,3 +6,13 @@ controls:
     (whitelist) policy to allow only authorized container images in the container
     platform.
   status: inherently met
+  artifact_description: |-
+    Supporting evidence is in the following documentation
+
+    https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html
+  status_justification: |-
```
Comment (rhmdnd) [inline controls/srg_ctr/SRG-APP-000386-CTR-000920.yml:13]: I didn't see this in the draft. Does it need to be there?

File: applications/openshift/scc/scc_limit_network_namespace/rule.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ references:
     nerc-cip: CIP-003-8 R6,CIP-004-6 R3,CIP-007-3 R6.1
     nist: CM-6,CM-6(1)
     pcidss: Req-2.2
-    srg: SRG-APP-000516-CTR-001325
+    srg: SRG-APP-000142-CTR-000330
```
Comment (rhmdnd) [inline applications/openshift/scc/scc_limit_network_namespace/rule.yml:29]: Just missed the reference initially?

File: controls/srg_ctr/SRG-APP-000386-CTR-000920.yml (hunk 1 above)
Comment (jhrozek) [reply controls/srg_ctr/SRG-APP-000386-CTR-000920.yml:13]: Umm but it is? Not sure if [this link](https://docs.google.com/spreadsheets/d/1ouRalvXzRHuhPPN_AKMvqqscUqGvVOkkEdMP3YYBO44/edit#gid=1236833936) would go straight to the cell, if not, row 149, column P

File: applications/openshift/scc/scc_limit_network_namespace/rule.yml (hunk 2 above)
Comment (jhrozek) [reply applications/openshift/scc/scc_limit_network_namespace/rule.yml:29]: `SRG-APP-000516-CTR-001325` is a catch-all control that says "Container platform components must be configured in accordance with the security configuration settings based on DoD security configuration or implementation guidance, including SRGs, STIGs, NSA configuration guides, CTOs, and DTMs.". The one used now is SRG-APP-000142-CTR-000330 which says Container images instantiated by the container platform must execute using least privileges. and I think that's a much better fit.

File: controls/srg_ctr/SRG-APP-000386-CTR-000920.yml (hunk 1 above)
Comment (rhmdnd) [reply controls/srg_ctr/SRG-APP-000386-CTR-000920.yml:13]: Aha - found it. Thanks

File: applications/openshift/scc/scc_limit_network_namespace/rule.yml (hunk 2 above)
Comment (rhmdnd) [reply applications/openshift/scc/scc_limit_network_namespace/rule.yml:29]: Ok - that makes sense.

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Pull request merged**